--- netcdf-fortran-4.6.1/fortran/netcdf_visibility.F90.orig	2024-12-17 20:13:21.135889021 +0100
+++ netcdf-fortran-4.6.1/fortran/netcdf_visibility.F90	2024-12-17 20:13:23.915873960 +0100
@@ -20,6 +20,7 @@
   public :: nf90_put_att, nf90_get_att
   
   ! Variable routines
-  public :: nf90_def_var, nf90_inq_varid, nf90_rename_var, nf90_inquire_variable 
+  public :: nf90_def_var, nf90_inq_varid, nf90_rename_var, nf90_inquire_variable, &
+            nf90_set_var_chunk_cache
   ! overloaded functions
   public :: nf90_put_var, nf90_get_var
